src/Components/DataviewItem/DataviewItem.tsx

```diff
@@ -2,12 +2,13 @@ import { Link } from 'react-router-dom';
 import classes from './Dataview.module.css';
 import { Person } from './types';
 import { useContext } from 'react';
-import { Context } from '../../pages/AppContainer';
+import { Context } from '../../context/context';
 
 function DataviewItem(props: Person) {
-  const sendDetailsQuery = useContext(Context);
+  const { getDetailsData } = useContext(Context);
+
   return (
-    <div className={classes.dataviewItem}>
+    <div className={classes.dataviewItem} data-testid="dataview-item">
```

Comment:
In future, better to avoid addition attributes only for testing. Right now, it's ok